remoulade/api/scheduler.py

```diff
@@ -0,0 +1,72 @@
+from functools import update_wrapper
+
+from flask import Blueprint, request
+from marshmallow import Schema, fields
+
+from remoulade import get_scheduler
+from remoulade.errors import NoScheduler
+from remoulade.scheduler import ScheduledJob
+
+scheduler_api = Blueprint("scheduler_api", __name__)
```

Comment:
```suggestion
scheduler_bp = Blueprint("scheduler", __name__, url_prefix="/scheduled")
```
And you can remove it for the routes below